spacetemplate/assets/agile.yaml

```diff
@@ -0,0 +1,387 @@
+        The component for this area of work.
+      required: no
+      type:
+        kind: string
+  child_types:
+  - *defectID
+  - *storyID
+
+- id: &storyID "6ff83406-caa7-47a9-9200-4ca796be11bb"
+  extends: *agileCommonTypeID
+  name: Story
+  can_construct: yes
+  description: 
+    A user story describes an executable chunk of a feature 
+    based on user experience.
+  icon: pficon pficon-image
+  fields:
+    "system.state":
+      label: State
+      description: The state of the story item.
+      required: yes
+      type:
+        simple_type:
+          kind: enum
+        base_type:
+          kind: string
+        values:
+        - New
+        - Open
+        - In Progress
+        - Resolved
+        - Closed
+    "effort":
+      label: Effort
+      description: 
+        The effort needed to implement this user story.
+      required: no
+      type:
+        kind: float
+    "acceptance_criteria": 
+      label: Acceptance criteria
+      description: 
+        The acceptance criteria describes rules for checking
+        if a chunk of work is completed.
+      required: no
+      type:
+        kind: markup
+    "component":
+      label: Component
+      description:
+        The component for this area of work.
+      required: no
+      type:
+        kind: string
+  child_types:
+  - *taskID
+  - *defectID
+
+- id: &taskID "2853459d-60ef-4fbe-aaf4-eccb9f554b34"
+  extends: *agileCommonTypeID
+  name: Task
+  can_construct: yes
+  description: 
+    An atomic task on the project.
+  icon: fa fa-tasks
+  fields:
+    "system.state":
+      label: State
+      description: The state of the Task.
+      required: yes
+      type:
+        simple_type:
+          kind: enum
+        base_type:
+          kind: string
+        values:
+        - New
+        - Open
+        - In Progress
+        - Resolved
+        - Closed
+
+- id: &defectID "fce0921f-ea70-4513-bb91-31d3aa8017f1"
+  extends: *agileCommonTypeID
+  name: Defect
+  can_construct: yes
+  description: 
+    This describes a defect in the system.
+  icon: fa fa-bug
+  fields:
+    "system.state":
+      label: State
+      description: The state of the defect.
+      required: yes
+      type:
+        simple_type:
+          kind: enum
+        base_type:
+          kind: string
+        values: 
+        - New
+        - Open
+        - In Progress
+        - Resolved
+        - Closed
+    "effort":
+      label: Effort
+      description: 
+        The effort needed to fix this defect.
+      required: no
+      type:
+        kind: float
+    "environment":
+      label: Environment
+      description:
+        The environment where this defect appears.
+      required: no
+      type:
+        kind: markup
+    "repro_steps":
+      label: Steps to reproduce
+      description: 
+        Steps to reproduce the defect.
+      required: no
+      type:
+        kind: markup
+    "severity":
+      label: Severity
+      description: TBD
```

Comment:
"The degree of impact that a defect has on the operation of the product or impact on security."
Ref: https://docs.google.com/presentation/d/1LFKTEHIiZ2KsiSWssgSh_a_yl2iiwWzwg0wKRuTroEk/edit#slide=id.g305b90d6b6_1_96

Reply:
Fixed.